nucleus/payara-modules/nucleus-microprofile/config-service/src/main/java/fish/payara/nucleus/microprofile/config/source/PasswordAliasConfigSource.java

```diff
@@ -86,6 +89,20 @@ public int getOrdinal() {
         return properties;
     }
 
+    @Override
+    public Set<String> getPropertyNames() {
+        Set<String> propertyNames = new HashSet<>();
+        if (store != null) {
```

Comment:
Unless I am missing something, this null check is unnecessary, please remove it

Reply:
I can be wrong, but if I remove this line and store is null, the line 96 will throw a NullPointerException

Reply:
store is an HK2 service singleton. It should never be null

Reply:
Ok. I'll remove this line. The same ocurrs in   getProperties() method (line 79)
https://github.com/payara/Payara/blob/19cec930b10c65ee02c7c2e000d17f5dec9bf5de/nucleus/payara-modules/nucleus-microprofile/config-service/src/main/java/fish/payara/nucleus/microprofile/config/source/PasswordAliasConfigSource.java#L79
And also in getValue() method (line 91)
https://github.com/payara/Payara/blob/19cec930b10c65ee02c7c2e000d17f5dec9bf5de/nucleus/payara-modules/nucleus-microprofile/config-service/src/main/java/fish/payara/nucleus/microprofile/config/source/PasswordAliasConfigSource.java#L91
Do I remove these lines too?

Reply:
Replace with `Objects.requireNotNul(name, "name should not be null)`
to do argument checking.